gensrc/script/functions.py

```diff
@@ -311,6 +311,9 @@
     [30410, 'parse_url', 'VARCHAR', ['VARCHAR', 'VARCHAR'], 'StringFunctions::parse_url',
      'StringFunctions::parse_url_prepare', 'StringFunctions::parse_url_close'],
     [30420, 'strcmp', 'INT', ['VARCHAR', 'VARCHAR'], 'StringFunctions::strcmp'],
+    
+    [30430, 'compress', 'VARBINARY', ['VARCHAR', 'VARCHAR'], 'StringFunctions::compress'],
+    [30431, 'compress', 'VARBINARY', ['VARCHAR'], 'StringFunctions::compress'],
```

Comment:
Default argument of compress can be handle in the same way as lpad/rpad,  Can you fill the default const argument if compress misses its 2nd argument? so 30431 is tedious and can be removed away.

AstBuilder.java
```java
        if (functionName.equals(FunctionSet.LPAD) || functionName.equals(FunctionSet.RPAD)) {
            if (context.expression().size() == 2) { 
                Expr e1 = (Expr) visit(context.expression(0));
                Expr e2 = (Expr) visit(context.expression(1));
                FunctionCallExpr functionCallExpr = new FunctionCallExpr(
                        fnName, Lists.newArrayList(e1, e2, new StringLiteral(" ")), pos);
                return functionCallExpr;
            }    
        }    

```